docs/00-introduction.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>I would add:</p><ul><li>You want to summarize data from your catalog</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/eoapi-fedgeoday25-workshop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
This is a cool thing you can do with a pgstac database - I'll add it to the pgstac notebook.